src/data/faq_de.json

```diff
@@ -48,14 +48,14 @@
                         ]
                     },
                     {
-                        "title": "Wie erstelle ich ein Fehlerbericht?",
+                        "title": "Wie erstelle ich einen Fehlerbericht?",
                         "anchor": "error_log",
                         "active": true,
                         "textblock": [
-                            "Mit Version 2.2 bietet die Corona-Warn-App (CWA) eine Funktion zur technischen Problem-Analyse. Fehlerberichte soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der technischen Hotline (national: 0800 7540001 bzw. international: +49 30 498 75401), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehlerberichte wird nur benötigt, wenn das CWA-Team danach fragt.",
-                            "<ol><li>In dem Fall öffnen Sie bitte ein Issue auf GitHub und beschreiben Sie das Problem.</li><li>Starten Sie die Aufzeichnung in der CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2//' target='_blank' rel='noopener noreferrer'>siehe Blog-Artikel</a>).<br>- Versuchen Sie, das fehlerhafte Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen.<br>- Ist das gelungen, senden Sie den Fehlerbericht an den CWA-Server. Die Daten werden dabei anonymisiert und verschlüsselt. Bei Interesse können Sie den Bericht auch lokal auf Ihrem Smartphone abspeichern.<br>- Als Rückmeldung erhalten Sie eine eindeutige Kennung, die Sie auch in der Liste aller IDs finden <br>- Stoppen Sie die Fehleranalyse.</li><li>Aktualisieren Sie das GitHub-Issue mit Ihrer eindeutigen Kennung vom Server, damit der aufgezeichnete Bericht Ihrem Issue zugeordnet werden kann.</li></ol>",
+                            "Mit Version 2.2 bietet die Corona-Warn-App (CWA) eine Funktion zur technischen Problem-Analyse. Fehlerberichte soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der technischen Hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehlerberichte wird nur benötigt, wenn das CWA-Team danach fragt.",

```

Comment:
"Ein Fehlerberichte" => "Ein Fehlerbericht"

Reply:
Thanks!
Corrected in https://github.com/corona-warn-app/cwa-website/pull/1267/commits/10e6b18722bfdc9fb5fd83f4830a5dfdf7a2e3c6.